src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -477,16 +392,25 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
-    status = uct_ib_mlx5_devx_check_rndv(md);
-    if (status != UCS_OK) {
+    md->zero_buf = ucs_memalign(ucs_get_page_size(), ucs_get_page_size(), "zero umem");
+    if (md->zero_buf == NULL) {
+        ucs_error("failed to allocate null zero buffer: %m");
```

Comment:
null zero buffer?

Reply:
it's not your code, but better define another error label and cleanup the mpool allocated on line 386